src/trufont/util/drawing.py

```diff
@@ -440,9 +440,14 @@ def drawLayerPoints(
             # others
             prev = points[-1]
             breakHandle = path.open
+            quadraticPath = any(point.type == "qcurve" for point in points)
```

Comment:
ugh. this whole code is written to read the path only once. why is this needed?

Reply:
It's a cosmetic thing mostly, I find drawn handles on qcurve off-curves confusing because they suggest curve semantics. Leave in instead? Do something else?

Reply:
it's not curve semantics, it's smooth point semantics

Reply:
I would leave as-is personally